docs/traffic-policy/actions/compress-response/config.mdx

```diff
@@ -0,0 +1,35 @@
+## Configuration Reference
+
+This is the [Traffic Policy](/docs/http/traffic-policy/index.mdx) configuration 
+reference for this action.
+
+### Action Type
+
+`compress-response`
+
+### Configuration Fields
+
+| Parameter    | Type              | Description                                                                                                                                       |
+| ------------ | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `algorithms` | `[]string` | The list of allowed compression algorithms to be considered during encoding type negotiation. Optional. Defaults to `gzip`, `deflate`, `br`, `compress`. |
```

Comment:
nit: no longer need "Optional" since we're explicitly calling out Required params